### PR TITLE
Correct the tool "sysctl" name

### PR DESCRIPTION
**Pull request by sunlintong**
Is the tool name "sysctls" should be "sysctl" ?

### CODE REVIEW

File: docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ the machine even if it is being drained of applications in preparation for a reb
 
 ### Node capacity
 
-The capacity of the node (number of cpus and amount of memory) is part of the node object.
+The capacity of the node (numbers of cpu and amount of memory) is part of the node object.
```
Comment (tengqm) [inline docs/concepts/architecture/nodes.md]: This should be left as is

File: docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ If you are using a guide involving Salt, see [Configuring Kubernetes with Salt](
 
 * [Using Admission Controllers](/docs/admin/admission-controllers/) explains plug-ins which intercepts requests to the Kubernetes API server after authentication and authorization.
 
-* [Using Sysctls in a Kubernetes Cluster](/docs/concepts/cluster-administration/sysctl-cluster/) describes to an administrator how to use the `sysctl` command-line tool to set kernel parameters .
+* [Using Sysctl in a Kubernetes Cluster](/docs/concepts/cluster-administration/sysctl-cluster/) describes to an administrator how to use the `sysctl` command-line tool to set kernel parameters .
```
Comment (tengqm) [inline docs/concepts/cluster-administration/cluster-administration-overview.md]: I don't think this is a valid change. The 'sysctls' here meant the various knobs an admin can tune, it is not about the command line tool.

Comment (sunlintong) [reply docs/concepts/cluster-administration/cluster-administration-overview.md]: yes, I didn't understand this well, thanks for your correct :P

File: docs/concepts/architecture/nodes.md (hunk 1 above)
Comment (steveperry-53) [reply docs/concepts/architecture/nodes.md]: Agree

File: docs/concepts/cluster-administration/cluster-administration-overview.md (hunk 2 above)
Comment (steveperry-53) [reply docs/concepts/cluster-administration/cluster-administration-overview.md]: Agree with @tengqm.

**Comment by steveperry-53** [comment]
@sunlintong Thanks for doing this. After you make the changes suggested by @tengqm, we should be able to merge this.

**Pull request closed without merging**